tests/integration/cli/tests/fixtures/init5.toml

```diff
@@ -0,0 +1,7 @@
+[package]
+name = "test-wasmer-init"
+version = "RANDOMVERSION1.RANDOMVERSION2.RANDOMVERSION3"
+edition = "2021"
+description = "description of package test-wasmer-init"
+
+[dependencies]
```

Comment:
This file shouldn't exist is a `cargo wapm` one which we agreed not to have in this PR

Reply:
Note: unless is used for testing the `cargo.toml` naming parsing into `wasmer init`

Reply:
I removed init3 and init5, we don't need them anymore.